scroll-animations-1/Overview.bs

```diff
@@ -498,6 +503,8 @@ spec: cssom-view-1; type: dfn;
 
 	If the 0% position and 100% position coincide
 	(i.e. the denominator in the [=current time=] formula is zero),
+	or if there is no corresponding [=scroll offset=]
```

Comment:
Doesn't no corresponding scroll offset imply this happens when there's no scrollable overflow - which is not strictly true? I was thinking we could just say the [=view progress visibility range=] is the scrollable overflow size and let the rest fall out from the math. That said, I'm okay with making timelines looking at the root scroll on printed media blanket inactive I just don't want this to apply to other cases where we have well established behaviors as the overflow becomes entirely visible, e.g. demo in https://github.com/w3c/csswg-drafts/issues/7778#issuecomment-1310360733

Reply:
> Doesn't no corresponding scroll offset imply this happens when there's no scrollable overflow - which is not strictly true?

I was expecting that any scroll container has to have a scroll offset, even if it's always zero.

> I was thinking we could just say the [=view progress visibility range=] is the scrollable overflow size and let the rest fall out from the math.

It becomes a little bit vague what that means for paged media, especially if the content is longer than the root element itself.